### ISSUE TITLE
improvement: relative links

### ISSUE THREAD

**Issue by Hendrikto** [daltonmenezes/aura-theme#187]
Hi Dalton,

What do you think about [making links relative](https://github.com/Hendrikto/aura-theme/tree/improvement/relative_links)?

If you look at the [main branch of my fork](https://github.com/Hendrikto/aura-theme/tree/main), you will see that all of the links to the ports etc. are linking to your personal repo, even for forks of Aura Theme. This means that it is harder to develop new ports and work on the repo, because one cannot easily test whether the links work. Also, it makes it harder to fork the project in general.

It would be nice if one could click on the links in the READMEs, and stay within the same project while doing so. I made the links for the Sway port relative, so this already works for that port on the [relative links branch](https://github.com/Hendrikto/aura-theme/tree/improvement/relative_links). One can click from the [main README](https://github.com/Hendrikto/aura-theme/tree/improvement/relative_links) through to the [Sway port README](https://github.com/Hendrikto/aura-theme/tree/improvement/relative_links/packages/sway), and then to [`aura-theme.conf`](https://github.com/Hendrikto/aura-theme/blob/improvement/relative_links/packages/sway/aura-theme.conf), all while staying within the fork.

In summary, making links relative would:

* change nothing for your own repo,
* fix links in other forks, and
* make it easier to develop ports.

**Comment by daltonmenezes** [comment]
Hi @Hendrikto , good idea! it looks great for me!

**Comment by Hendrikto** [comment]
Great :)

Work is already ongoing, but it is a bit more effort than I anticipated, as I have to test whether all links still work.

**Comment by Hendrikto** [comment]
I am mostly done with the conversion to relative links. As mentioned above, it was more work than anticipated, as I had to review everything. I found several problems with some ports, which I fixed. I need to review the fixed ports, and some other minor things. Here is a progress report:

# Review

## General

* [ ] PR template
* main README
  * Aura
    * [X] Color Palettes
    * [X] Wallpapers
    * [X] Shop
  * Documentation
    * [X] Creating Ports
    * [X] CLI
    * [X] API
    * [X] Publishing Ports

## Documentation

* [X] Creating Ports
* [X] CLI
* [X] API
* [ ] Publishing Ports

## Ports

* [ ] alacritty
* [X] chrome
* [ ] code-sandbox
* [X] color-palettes
* [ ] fig
* [X] firefox
* [ ] gnome-terminal
* [X] hyper
* [X] icue
* [X] insomnia
* [ ] iterm
* [ ] kitty
* [ ] konsole
* [ ] mattermost
* [X] ngenuity
* [X] plasma
* [X] rxvt-unicode
* [X] slack
* [ ] sublime-text
* [X] sway
* [X] tabby
* [ ] telegram
* [ ] terminal-app
* [ ] termux
* [ ] tilix
* [X] vivaldi
* [X] vscode
* [X] wallpapers
* [X] waybar
* [ ] windows-terminal
